core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md

```diff
@@ -0,0 +1,23 @@
+---
+description: >-
+  The Appsmith platform allows us to share data across pages either via query params or via local storage.
+---
+
+# Data Sharing Across Pages
+
+## Sharing data via query params 
+
+We can pass query params to a given page by adding the query params object to `NavigateTo` Action. To do this click on the JS mode for Action and update it as follows.
```

Comment:
Can you change the sentences to passive voice everywhere on the page? 
i.e "Query params can be passed by adding..."
instead "of we can pass ...."

Reply:
Done